add 2023-05-25 URTI progress report

mossmann opened the pull request:
phase one completion

mossmann committed:
add 2023-05-25 URTI progress report

(phase one completion)

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+## Architecture
+
+We plan to build URTI as a system comprised of two printed circuit board assemblies: a mainboard, and a user interface board. The mainboard will include the radio section and a USB interface and will be capable of serving as a Software-Defined Radio peripheral with or without the user interface board attached. The user interface board will include tactile, controls, display, application processor ...
```
remove the comma after mainboard
also, it's not clear what the adjective tactile is describing in the last sentence. If the adjective tactile is meant to describe the controls, we should remove the comma after tactile.

epenelope replied:
The second sentence is a little bit hard to follow. I suggest splitting it into two sentences:
"The mainboard will include the radio section and a USB interface. It will be capable of serving as..."

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+### Quadrature Transmitter
+
+We will use [MAX2831](https://www.analog.com/en/products/max2831.html) or [MAX2830](https://www.analog.com/en/products/max2830.html) to convert quadrature analog baseband signals to an Intermediate Frequency (IF) between 2.4 GHz and 2.5 GHz for transmission. MAX2831 and MAX2830 are both very similar to MAX2837 used on HackRF One, but we chose to test the components individually.
```
I think you could explain why we selected those two components for evaluation. Reading this, it sounds like you are saying MAX2837 was also a candidate for URTI. We know why it wasn't a candidate, but the audience of this report might not have that context.

And why did we choose to test them individually? Is it because they were dissimilar enough that we couldn't rely on our familiarity with MAX2837? 

Finally, it's not clear what the word 'individually' means in the last sentence. Does it mean we tested each of those two components separately from MAX2837?

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+We will use [MAX2120](https://www.analog.com/en/products/max2120.html) to convert an IF between 925 MHz and 2175 MHz to quadrature analog baseband. MAX2120 is somewhat similar to MAX2837 used on HackRF One, but it is a receiver, not a transceiver. It includes a quadrature downconverter, a frequency synthesizer, baseband filters, and gain.
+
+We used [MAX2223EVKIT](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/max2223evkit.html), an off-the-shelf evaluation board, to test [MAX2223](https://www.analog.com/en/products/max2223.html) and MAX2120. We controlled the device with a GreatFET One and connected the RF ports to an RF signal generator and spectrum analyzer. MAX2223 is similar but l ...
```
The sentence "MAX2223 is similar but lacks baseband filters" doesn't make sense to me in this paragraph. What is it similar to?

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+RF gain control worked, but it requires an analog voltage generated by a DAC. There is an additional 15 dB of digitally controlled baseband gain, but it seemed to amplify the noise floor just as much as the signal. We may pick a fixed setting to match the ADC and then adjust only the RF gain.
+
+We tested MAX2223 because it was the component initially installed on the evaluation board, but we plan to use MAX2120 because of its analog baseband filter and low cost.
```
This seems out of place; I think this information belongs in line 87, where you first mention the component we chose.

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+We used [MAX2223EVKIT](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/max2223evkit.html), an off-the-shelf evaluation board, to test [MAX2223](https://www.analog.com/en/products/max2223.html) and MAX2120. We controlled the device with a GreatFET One and connected the RF ports to an RF signal generator and spectrum analyzer. MAX2223 is similar but l ...
+
+Both components worked as expected. We observed that they are capable of tuning well outside their advertised ranges. MAX2223 operated from 513 MHz to 2653 MHz while MAX2120 operated from 499 MHz to 2442 MHz.
```
The statement "both components worked as expected" seems to conflict with a statement in the following paragraph, "there are several other errors and omissions in the data sheet". If they worked as expected, what did we base our expectations on if not the datasheet?

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+### SMA Connector and Trace Width
+
+Because of our need to design many small PCBs for testing RF components and sub-circuits throughout the project, it was important to establish a baseline test PCB design featuring RF traces with 50 ohm characteristic impedance and low-loss RF connectors at a reasonable cost. [Initial tests](../../rf-testing/2022-04-06-sma-and-trace-width-test) of PCBs manufactured by [JLCPCB](https://jlcpcb.com/) ...
```
Replace the first sentence with: "Establishing a baseline test PCB design featuring RF traces with 50 ohm characteristic impedance and low-loss RF connectors at a reasonable cost was important because we will need to design many small PCBs for testing RF components and sub-circuits throughout the project."

epenelope replied:
Replace "arrive upon a solution" with "arrive at a solution"

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+HackRF One supports arbitrary sample rates by producing the sample clock with the very flexible Si5351. In contrast, URTI will be able to perform resampling in its FPGA, making it possible to use a fixed sample rate for the ADC and DAC.
+
+We plan to generate a single 40 MHz clock signal that is used as the ADC/DAC sample clock and also as a reference clock for each of the RF synthesizers. The mixers (RFFC5072) and quadrature transmitters (MAX2830 and MAX2831) support a 40 MHz reference clock directly. The quadrature receiver (MAX2120) can accept a 20 MHz reference clock which we will generate by dividing the sample clock by two wi ...
```
Change the first sentence to "We plan to generate a single 40 MHz clock signal that is used as the ADC/DAC sample clock and as a reference clock for each RF synthesizer. "

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+We plan to generate a single 40 MHz clock signal that is used as the ADC/DAC sample clock and also as a reference clock for each of the RF synthesizers. The mixers (RFFC5072) and quadrature transmitters (MAX2830 and MAX2831) support a 40 MHz reference clock directly. The quadrature receiver (MAX2120) can accept a 20 MHz reference clock which we will generate by dividing the sample clock by two wi ...
+
+In the initial mainboard prototype we plan to use a single 40 MHz temperature-compensated crystal oscillator (TCXO). In the final mainboard design we will likely use a voltage-controlled crystal oscillator (VCXO) capable of being disciplined by an external clock input.
```
Change the first sentence to: "We plan to use a single 40 MHz temperature-compensated crystal oscillator (TCXO) in the initial mainboard prototype."

Add a comma after the word 'design' in the second sentence

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+Although a limiter does not appear in the preliminary block diagram, we think that we will want to include one or more to protect RF components in the final mainboard design.
+
+Modern RF ICs of the sort we expect to use in URTI and related designs such as [Amalthea](https://github.com/greatscottgadgets/amalthea-hardware) often specify a maximum input power of 10 dBm to 20 dBm. Unfortunately, off-the-shelf RF limiter components and most [traditional designs](https://www.skyworksinc.com/-/media/SkyWorks/Documents/Products/1-100/200480C.pdf) do not begin to have an affect  ...
```
Change 'affect' to 'effect'

epenelope replied:
Change "The very wide operating frequency range and low cost requirements of URTI make this problem even more difficult to solve." to "URTI's wide operating frequency range and low-cost requirements make this problem even more challenging to solve."

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+Lim's key insight is that a Schottky diode anti-parallel to a PIN diode can be used to significantly reduce the limiting threshold of the PIN diode. Broadcom/Avago produced [ASML-5829](https://www.digikey.ca/en/articles/how-and-why-to-use-pin-diodes-for-rf-switching) and other PIN-Schottky limiters in a single package, but they are now obsolete. A few other PIN-Schottky limiters such as [SKY16602 ...
+
+Another trick used by Lim is the use of the parasitic capacitance and inductance of the limiter diodes as elements in a low-pass filter (which can have a high cutoff frequency when using small diodes). This improves the impedance match in the passband, reducing loss. The earliest reference we found to this technique for RF limiters was in HP/Agilent Application Note 1050, *Low Cost Surface Mount  ...
```
Change "used by Lim is the use of" to "Lim uses is applying"

epenelope replied:
To clarify the antecedent, add a word: "This _technique_ improves the impedance match in the passband..."

mossmann replied:
I'm replacing "Another trick used by Lim is the use of" with "Lim also suggests using".

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+## Progress on Overlapping Phases
+
+During phase one we additionally made progress on phases four and eight. Although completion of the later phases is still a long way off, work toward those software development goals in parallel with hardware design tasks will speed completion of software after hardware is available. This approach has allowed us to involve more team members in the project.
```
Add a comma after 'phase one'

epenelope replied:
Change 'work' to 'working'

epenelope commented on project-reports/2023-05-25-urti-progress-report/README.md:
```diff
@@ -0,0 +1,163 @@
+
+During phase one we additionally made progress on phases four and eight. Although completion of the later phases is still a long way off, work toward those software development goals in parallel with hardware design tasks will speed completion of software after hardware is available. This approach has allowed us to involve more team members in the project.
+
+Separate from URTI but partly in an effort to provide a platform for URTI developers, we have designed and assembled [Amalthea](https://github.com/greatscottgadgets/amalthea-hardware), a Software-Defined Radio add-on for Cynthion. Amalthea provides both transmit and receive capabilities with the same Cynthion-based FPGA back-end as planned for URTI, using our [LUNA](https://github.com/greatscottg ...
```
Replace 'in an effort to' with just 'to'

epenelope reviewed (changes requested):
Great start! I've commented in line with specific suggestions for refining edits.

epenelope reviewed (changes requested):
Great start! I've commented in line with specific suggestions for refining edits.

mossmann committed:
edit for readability and clarity

mossmann committed:
add DPDT switch test to URTI progress report

mossmann committed:
update URTI progress report date

mossmann commented:
Thank you for the great edits, @epenelope! I've addressed all your suggestions, added a short section on DPDT RF switch testing, and updated the report date.

epenelope reviewed (approved):
This looks great; thanks, @mossmann !
